peerdid/peer_did.py

```diff
@@ -62,7 +104,7 @@ def save_peer_did(peer_did: PEER_DID, storage: Optional[Storage] = None):
     :raise ValueError: raises if peer_did parameter does not match peer_did spec
     """
     if is_peer_did(peer_did=peer_did):
-        storage = storage or FileStorage()
+        storage = storage or FileStorage(peer_did_filename='new peerDID')
```

Comment:
I think the file name should be a peer DID itself, so that we can find it in `resolve` easily.